jenkins/Jenkinsfile.j2

```
@@ -933,4 +984,36 @@ stage('Deploy') {
       }
     }
   }
+  // if (env.BRANCH_NAME == 'main' && rebuild_docker_images) {
+  if (env.BRANCH_NAME == 'PR-11329' && rebuild_docker_images && upstream_revision != null) {
+    node('CPU') {
+      ws({{ m.per_exec_ws('tvm/deploy-docker') }}) {
+        try {
+          withCredentials([string(
+            credentialsId: 'dockerhub-tlcpackstaging-key',
+            variable: 'DOCKERHUB_KEY',
+          )]) {
+            sh(
+              script: 'docker login -u tlcpackstaging -p ${DOCKERHUB_KEY}',
+              label: 'Log in to Docker Hub',
+            )
+          }
+          def date_Ymd_HMS = sh(
+            script: 'python -c \'import datetime; print(datetime.datetime.now().strftime("%Y%m%d-%H%M%S"))\'',
+            label: 'Determine date',
+            returnStdout: true,
+          ).trim()
+          def tag = "${date_Ymd_HMS}-${upstream_revision.substring(0, 8)}"
+          {% for image in images %}
+          update_docker({{ image.name }}, "tlcpackstaging/test_{{ image.name }}:${tag}")
```

I think this is literally for testing, but I'll leave a comment just in case 😸 

```suggestion
          update_docker({{ image.name }}, "tlcpackstaging/{{ image.name }}:${tag}")
```